zerver/openapi/zulip.yaml

```diff
@@ -14059,6 +14059,21 @@ paths:
                     - 2
                     - 3
                   example: 1
+                week_start_day:
```

Comment:
This commit should be squashed with the second commit, which does the API changes as the documentation and test changes should be in the same commit that does code changes.